apps/server/lib/google-drive/src/internal/to-file.ts

```
@@ -0,0 +1,147 @@
+	) {
+		parts.push(value);
+	} else if (isBlobLike(value)) {
+		parts.push(value instanceof Blob ? value : await value.arrayBuffer());
+	} else if (
+		isAsyncIterable(value) // includes Readable, ReadableStream, etc.
+	) {
+		for await (const chunk of value) {
+			parts.push(...(await getBytes(chunk as BlobLikePart))); // TODO, consider validating?
+		}
+	} else {
+		const constructor = value?.constructor?.name;
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)